changelog/fragments/bump-opm-version.yaml

```diff
@@ -0,0 +1,16 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Bumping OPM version from v1.19.1 to v1.21.0 to allow for ppc64le and s390x support
```

Comment:
```suggestion
      Bumping OPM version from v1.19.1 to v1.21.0. For further information about this changes see: https://github.com/operator-framework/operator-registry/releases
```

Reply:
@gquillar, 

Thank you for the contribution 🥇 

Can we also update the title? 
Bumping OPM version from v1.19.1 to v1.21.0 (one of its motivations is to provide support for  ppc64le and s390x arches)

Reply:
I've updated the title. I will also commit your suggestion.